{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -9,3 +9,6 @@ test = pytest
 omit =
     */__main__.py
     */test/*
+
+[flake8]
+exclude = setup.py, {{ cookiecutter.project_slug}}/__init__.py
```

Comment:
What's the reason for excluding `__init__.py`?

Reply:
Because the line for the version number is too long and it looks terrible if you make it 80 characters, so I'm just ingnoring the file

Reply:
OK. I think for the most part we can either ignore the 80 char rule or set it to be longer before it complains.